standard_template/standard/clause_6_informative_text.adoc

```diff
@@ -262,3 +276,231 @@ If multiple aliases match an attribute name, this is an error condition, the dec
+NetCDf-Classic-LD explicitly includes all reference statements, even where the broadcast relationship can be inferred, for clarity and to aid comprehension.
+
+A reshape array has the same total number of elements as the original array, but includes extra dimensions, of size 1, defining the order which the extensive dimensions are handled in.
+
+To hold this information, netCDF-Classic-LD creates a new entity within the graph, representing this relationship, a `<$referenceEntity>`.
+
+This `<$referenceEntity>` is referenced by the `<$parentEntity>` using the predicate `<http://binary-array-ld.net/latest/references>`, i.e.:
+
+----
+  <$parentEntity> <http://binary-array-ld.net/latest/references> <$referenceEntity>
+----
+
+The `<$referenceEntity>` is defined to be of type `<http://binary-array-ld.net/latest/Reference>` and is expected to carry a single reference to the child entity in the relationship and a `bald:childBroadcast` statement defining the expansion of the child array to the same shape as the parent. i.e.:
+
+----
+  <$referenceEntity> a <http://binary-array-ld.net/latest/Reference> ;
+      <http://binary-array-ld.net/latest/childBroadcast> ($d1 $d2 $d3) ;
+      <http://binary-array-ld.net/latest/array> <$childEntity> .
+----
+
+`$dn` are all defined to be integers.  The object of this statement is an RDF list.
+
+There is an optional statement predicate available: `<http://binary-array-ld.net/latest/childBroadcast>`, for the rare cases where both parent and child arrays have to be broadcast to a common shape. 
+
+===== Mismatched reference 
+
+Where metadata schemes provide further variable reference predicates, it is possible for variable to variable references to be defined where a broadcast relationship does not exist.
+
+Applications may treat mismatches between reference definitions and the ability to broadcast as warning conditions, and skip the creation of statements, or as error conditions, and fail to create a graph.  This is an implementation detail.
+
+==== NetCDF Coordinate Variables
+
+NetCDF defines a special type of variable, called a Coordinate Variable, which is identified by being one-dimensional and having the same name as the single dimension used to size the variable.
+
+NetCDF-Classic_LD interprets Coordinate Variables as a case of variable referencing and includes entries as `bald:references` statements. 
+
+
+=== Worked Example
+
+Here the definition of a netCDF file, in CDL, with all data array elements set as missing, is presented. It is followed by an RDF graph interpretation of the netCDF, illustrating many of the interpretation features desribed in this chapter.
+
+----
+netcdf multi_array_reference.cdl {
+dimensions:
+	pdim0 = 11 ;
+	pdim1 = 17 ;
+variables:
+        int prefix_list(pdim0, pdim1) ;
```

Comment:
why is this prefix variable defined extensively, with respect to pdim0 and pdim1